src/Files.App/Views/Pages/PropertiesGeneral.xaml.cs

```diff
@@ -104,7 +105,30 @@ async Task<bool> SaveBaseAsync()
 						UIFilesystemHelpers.SetHiddenAttributeItem(item, ViewModel.IsHidden, itemMM)
 					);
 				}
+                
+				// Handles run as administrator for shortcuts
+				if (item.IsShortcut)
+				{
+					var shortcutItem = (ShortcutItem)item;
+
+					App.Logger.Warn("Is a shortcut file");
+
+					 var isApplication = !string.IsNullOrWhiteSpace(shortcutItem.TargetPath) &&
+						(shortcutItem.TargetPath.EndsWith(".exe", StringComparison.OrdinalIgnoreCase)
+						|| shortcutItem.TargetPath.EndsWith(".msi", StringComparison.OrdinalIgnoreCase)
+						|| shortcutItem.TargetPath.EndsWith(".bat", StringComparison.OrdinalIgnoreCase));
```

Comment:
This would be a good candidate for #10831. @QuaintMako

Reply:
Yes, it would be nice to have an extension method for IsExecutable

Reply:
Should I implement it in this pr?

Reply:
@gave92 opened a PR for this so let's wait for that to be merged and then you can take advantage of his changes.

Reply:
Ok